tests/sota_tools/auth_test.cc

```diff
@@ -50,6 +79,8 @@ TEST(authenticate, invalid_file) {
 #ifndef __NO_MAIN__
 int main(int argc, char **argv) {
   ::testing::InitGoogleTest(&argc, argv);
+  TestHelperProcess server_process("fake_http_server/ssl_server.py", "");
+  sleep(2);
```

Comment:
Can we use getFreePort() from test_utils.h to make this pick a port automatically?

Reply:
It is not so easy, because url with port number is inside auth archive.

Reply:
OK